packages/application-shell/src/components/handle-apollo-errors/handle-apollo-errors.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { QueryResult } from 'react-apollo';
+import { handleActionError } from '@commercetools-frontend/actions-global';
+import getDisplayName from '../../utils/get-display-name';
+
+type QueryResultNames = string[];
+type Props = {
+  [key: string]: QueryResult;
+};
+
+function handleApolloErrors(queryResultNames: QueryResultNames) {
```

Comment:
When should we use this again? :)

Reply:
I think we should deprecate it and eventually remove it. I don’t think anyone is using this 😅